docs/source/index.rst

```diff
@@ -36,6 +36,36 @@ user:
     print(kennethreitz.followers_count)
 
     followers = [str(f) for f in gh.followers('kennethreitz')]
+    
+    # create a repository
```

Comment:
Thinking now about how code comments should explain *why* more than *what*, is the method name `create_repository` not sufficient to explain *what* is happening here?

Reply:
Further, `create_repository` returns the created repository. Do we then need to re-retrieve it below?